pkg/deploy/api/v1/conversion.go

```diff
@@ -143,24 +206,6 @@ func convert_v1_DeploymentTriggerImageChangeParams_To_api_DeploymentTriggerImage
 	if err := s.Convert(&in.From, &out.From, 0); err != nil {
 		return err
 	}
-	switch in.From.Kind {
```

Comment:
Do we know for sure it's okay to stop supporting ancient triggers stored with those kinds?